perception/object_recognition/tracking/multi_object_tracker/src/multi_object_tracker_core.cpp

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright 2018 Autoware Foundation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *
+ * v1.0 Yukihiro Saito
+ */
+
+#include "multi_object_tracker/multi_object_tracker_core.hpp"
+#include <rclcpp_components/register_node_macro.hpp>
+
+#include <tf2_ros/create_timer_ros.h>
+#include <tf2_ros/create_timer_interface.h>
+#define EIGEN_MPL2_ONLY
+#include <Eigen/Core>
+#include <Eigen/Geometry>
+
+#include <string>
+
+MultiObjectTracker::MultiObjectTracker(const rclcpp::NodeOptions & node_options)
+: rclcpp::Node("multi_object_tracker", node_options)
+{
+  // Create publishers and subscribers
+  dynamic_object_sub_ =
+    create_subscription<autoware_perception_msgs::msg::DynamicObjectWithFeatureArray>(
+      "input", rclcpp::QoS{1},
```

Comment:
This had `latch = true` before, if this behavior should be kept, this would need the `transient_local` durability QoS.

Reply:
Ah good catch, I missed this. Changed in 0febc64